src/main/io/vtx_smartaudio.c

```diff
@@ -773,13 +870,23 @@ void vtxSASetPitMode(uint8_t onoff)
     return;
 }
 
+void vtxSASetFreq(uint16_t freq)
+{
+    if (saValidateFreq(freq)) {
+        saSetMode(0);        //need to be in FREE mode to set freq
+        saSetFreq(freq);
+    }
+}
+
 bool vtxSAGetBandAndChannel(uint8_t *pBand, uint8_t *pChannel)
 {
     if (!vtxSAIsReady()) {
         return false;
     }
 
-    *pBand = (saDevice.channel / 8) + 1;
+    // if in user-freq mode then report band as zero
+    *pBand = (saDevice.mode & SA_MODE_GET_FREQ_BY_FREQ) ? 0 :
+                                               ((saDevice.channel / 8) + 1);
```

Comment:
there's quite a few magic `8`'s added in this PR that should be using the appropriate #defines for the CHANNEL count.

Reply:
The '8's are coming from code copied from the original version.  I think it's more about the data format as read from the hardware, so I don't think using VTX_RTC6705_CHAN_COUNT would be appropriate there.  Maybe a good place for a macro...